_pages/04-data.md

```diff
@@ -115,3 +115,19 @@ The M-Lab *test name* Data Set, *date range used*. *M-Lab test URL*
 For example:
 
 The M-Lab NDT Data Set 2009-02-11–2015-12-21. [https://measurementlab.net/tests/ndt]({{ site.baseurl }}/tests/ndt)
+
+or, in BibTeX:
```

Comment:
Could this say "BibTeX format" and be a link to a resource about BibTex?

Reply:
"BibTeX format" will definitely make that change.

As far as a link to "What is BibTeX?", I'd probably just link to Wikipedia, which feels a little like saying "look it up yourself if you are confused".

Reply:
that's what I usually do- link to wikipedia. which isn't presumptive, but helpful in saving the person who would need to ask that from needing to do the search.